docs/introduction.rst

```diff
@@ -13,92 +13,96 @@ or modifying files produced by other programs (e.g.,
+
+At the end of the simulation we are typically only interested in the genetic
+relationships between the nodes in those individuals which are still alive;
+other parts of the genealogy are irrelevant. To save having to store this unnecessary
+genealogical information, SLiM *simplifies* the tree sequence as it goes along, retaining
+only certain parts of the genetic genealogy. When the tree sequence is output, the result
+then looks something like the situation on the right, in which many of the nodes and
+individuals have been removed.
 
 .. figure:: _static/pedigree01.png
    :scale: 40%
 
+.. _sec_left_in_tree_sequence:
 
 *************************************
 Who and what is in the tree sequence?
 *************************************
 
-OK, who and what exactly *is* left in the tree sequence?
-We clearly want all individuals of the *final* generation
-to have their genealogies recorded in the tree sequence.
-in the picture above there is one additional individuals pictured
-in the tree sequence that was not alive at the end of the simulation,
-because it was *remembered*, using the ``treeSeqRememberIndividuals()`` method.
-
+OK, who and what exactly *is* left in the tree sequence after the unnecessary
+information has been removed? Here's the diagram above with some added terminology.
 
 .. figure:: _static/pedigree2.png
    :scale: 40%
-
-The nodes (i.e., the chromosomes) of these individuals
-are the *samples* of the tree sequence.
-Saying that a chromosome is a *sample* implies that we have full information about
-its genealogy, and many operations on tree sequences act on the samples
-by default.
-See the `tskit data model <https://tskit.readthedocs.io/en/latest/data-model.html#nodes-genomes-or-individuals>`_ for more on this topic.
-The easiest way to pull out individuals alive at a particular time
-is with the :meth:`.SlimTreeSequence.individuals_alive_at()` method:
-see :ref:`sec_extracting_individuals` for more discussion.
-
-Here's a reminder of the terminology:
-
-
-.. figure:: _static/pedigree3.png
-   :scale: 40%
-
-.. note::
-
-   In previous versions of SLiM/pyslim, the *first generation* of individuals
-   were retained in the tree sequence, to allow recapitation.
-   This is no longer necessary, so these are *no longer present*, unless
-   you specifically Remember them -- see the :ref:`sec_tutorial_recapitation`
-   section of the tutorial for more discussion of how this works.
+   :align: right
+
+In the recorded tree sequence the alive individuals, which in this model are those
+in the final generation, have all had their nodes marked as *samples*. As
+you may recall, this means we have their full genetic ancestry. The sample nodes,
+and the individuals containing them, are always present in the tree sequence, which
+should be clear in the diagram. (Note that additional nodes can be
+:ref:`marked as samples <sec_remembering_individuals>` as detailed in the tutorial.)
```

Comment:
I'd end this paragraph with "The sample nodes, and the individuals containing them, are always present in the tree sequence." (the rest is a bit confusing at this point).

Reply:
Good point, I was probably going too deep too early.